packager/signer/signer_test.go

```diff
@@ -1026,3 +1026,24 @@ func (this *SignerSuite) TestPrometheusMetricGatewayRequestsLatency() {
 	}
 
 }
+
+func (this *SignerSuite) TestIfCappedDontSignAndProxyFullDocument() {
+	urlSets := []util.URLSet{{
+		Sign: &util.URLPattern{[]string{"https"}, "", this.httpsHost(), stringPtr("/amp/.*"), []string{}, stringPtr(""), false, 2000, nil}}}
+
+	const uncappedTailLength = 100
+	veryLongArray := [maxBodyLength + uncappedTailLength]string{}
+	veryLongString := strings.Join(veryLongArray[:], "a")
+	var customFakeBody = []byte("<html amp><body>" + veryLongString)
+
+	this.fakeHandler = func(resp http.ResponseWriter, req *http.Request) {
+		resp.Header().Set("Content-Type", "text/html")
+		resp.Write(customFakeBody)
+	}
+	resp := this.get(this.T(), this.new(urlSets), "/priv/doc?sign="+url.QueryEscape(this.httpsURL()+fakePath))
```

Comment:
This code doesn't look up to date.  In particular, I remember replacing this.get() during the FIXIT in PR https://github.com/ampproject/amppackager/commit/ec46ffd61435d435b29ac5a95bad63ef65b2a9cc

Reply:
Thanks, fixed.